projects/ui-policy-studio-angular/src/lib/components/flow-details-phase-step/gio-ps-flow-details-phase-step.component.html

```diff
@@ -25,6 +25,9 @@
     <span *ngIf="genericPolicy?.type === 'SHARED_POLICY_GROUP'" class="gio-badge-neutral" matTooltip="Shared Policy Group">
       <mat-icon svgIcon="gio:users"></mat-icon>
     </span>
+    @if (getPrerequisiteMessage) {
+      <mat-icon class="info__head__info" svgIcon="gio:info" [matTooltip]="getPrerequisiteMessage"></mat-icon>
```

Comment:
will it be good to have a tooltip with a potential big text ?

Reply:
I don't really have anything better :/ and that's what figma wants . I'll add it in the description part of the catalog too.